crates/miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -479,7 +479,7 @@ end
 #! slot.
 #!
 #! Inputs:  [slot_id_prefix, slot_id_suffix, KEY, NEW_VALUE, pad(6)]
-#! Outputs: [OLD_MAP_ROOT, OLD_MAP_VALUE, pad(8)]
+#! Outputs: [OLD_VALUE, pad(8)]
```

Comment:
Can you address this comment as well https://github.com/0xMiden/miden-base/issues/2147#issuecomment-3630664320? Or do you think a different PR is better?